extension/amqp/publisher_test.go

```diff
@@ -60,9 +59,9 @@ func TestNotificationPublisher_Publish(t *testing.T) {
 	})
 }
 
-func getLogger() (goengine.Logger, *test.Hook) {
-	logger, loggerHook := test.NewNullLogger()
-	logger.SetLevel(logrus.DebugLevel)
-
-	return goengineLogger.Wrap(logger), loggerHook
-}
+//func getLogger() (goengine.Logger, *test.Hook) {
```

Comment:
Why did we comment this block? Can it be removed completely?

Reply:
Missed it 🤦‍♂